docker-compose.override.yml

```diff
@@ -21,29 +21,16 @@ services:
       - traefik.tags=${TRAEFIK_TAG}
     environment:
       - API_URL=https://${DOMAIN}/api/v1/swagger/
+    ports:
+      - '8080:8080'
   proxy:
-    image: traefik:v1.5
+    image: jwilder/nginx-proxy:alpine
```

Comment:
I'd suggest that we maintain the nginx config file in this repo; the one here seems like it might not exactly match our needs.

Reply:
@briandds @jgarberdds Do you guys want to sync up on this, so any changes can be made and the PR completed?